com.nickgirga.webready.yml

```diff
@@ -0,0 +1,49 @@
+id: com.nickgirga.webready
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: webready
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # access to home files; for locating WADs and custom installed source ports
```

Comment:
What's the relation with this comment and what the app seems to do?

Reply:
Sorry. I used an old manifest of mine for another app as a template. Forgot to update the comment. The permission is needed for this app. Will update it.